.github/workflows/build.yml

```
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@v3
```

Can we upgrade to v4
```suggestion
      - uses: actions/checkout@v4
```

---

Done